src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -214,15 +214,27 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
     private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberString, String responseNumberString) {
         StringBuilder tableHeaderFragmentHtml = new StringBuilder();
         String tableHeaderFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_HEADER_FRAGMENT;
-
-        for(int i = 0 ; i < numOfRubricChoices ; i++) {
-            String tableHeaderCell = 
-                    FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
-                            "${qnIndex}", questionNumberString,
-                            "${respIndex}", responseNumberString,
-                            "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", rubricChoices.get(i));
-            tableHeaderFragmentHtml.append(tableHeaderCell + Const.EOL);
+        
+        if(getChoiceAssginedWeight(rubricChoices) == null){
+            for(int i = 0 ; i < numOfRubricChoices ; i++) {
+                String tableHeaderCell = 
+                        FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
+                                "${qnIndex}", questionNumberString,
+                                "${respIndex}", responseNumberString,
+                                "${col}", Integer.toString(i),
+                                "${rubricChoiceValue}", (rubricChoices.get(i) + "(" + (numOfRubricChoices-i) + ")"));
+                tableHeaderFragmentHtml.append(tableHeaderCell + Const.EOL);
+                }
+        }else{
+            for(int i = 0 ; i < numOfRubricChoices ; i++) {
+                String tableHeaderCell = 
+                        FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
+                                "${qnIndex}", questionNumberString,
+                                "${respIndex}", responseNumberString,
+                                "${col}", Integer.toString(i),
+                                "${rubricChoiceValue}", rubricChoices.get(i));
+                tableHeaderFragmentHtml.append(tableHeaderCell + Const.EOL);
+                }
```

Comment:
The coding style is off in a few aspects here:
- `if (condition) {` and `for (loop) {` notice the space before/after the brackets
- `} else {` ditto
- `(a - b)` ditto, before/after the minus
- `int i = 0; i < ...; i++` no whitespace before the semicolon
- The closing `}` should be on the same indentation level as the starting `for` keyword.